modules/ibi-create-config-iso.adoc

```diff
@@ -0,0 +1,217 @@
+kind: ClusterImageSet
+metadata:
+  name: ibi-img-version-arch <1>
+spec:
+  releaseImage: ibi.example.com:path/to/release/images:version-arch <2>
+----
+<1> Specify the name for the `ClusterImageSet` resource.
+<2> Specify the address for the release image to use for the deployment. 
+// Does this need to sync with the version used in the install or any requirements here?
+
+.. Create the `ClusterImageSet` resource by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f ibi-cluster-image-set.yaml
+----
+
+. Create the `ImageClusterInstall` resource:
+
+.. Create a YAML file that defines the `ImageClusterInstall` resource:
++
+.Example `ibi-image-cluster-install.yaml` file
+[source,yaml]
+----
+apiVersion: extensions.hive.openshift.io/v1alpha1
+kind: ImageClusterInstall
+metadata:
+  name: ibi-image-install <1>
+  namespace: ibi-test
+spec:
+  bareMetalHostRef:
+    name: ibi-bmh <2>
+    namespace: openshift-machine-api
+  clusterDeploymentRef:
+    name: ibi-cluster-deployment <3>
+  hostname: ibi-host <4>
+  imageSetRef:
+    name: ibi-image-install <5>
+  nodeIP: 192.168.112.24 <6>
+----
+<1> Specify the name for the `ImageClusterInstall` resource.
+<2> Specify the `BareMetalHost` resource that you want to target for the image-based installation.
+<3> Specify the name of the `ClusterDeployment` resource that you want to use for the image-based installation of the target host.
+<4> Specify the host name for the cluster.
+<5> Specify the name of the `ClusterImageSet` resource you used to define the container release images to use for deployment.
+<6> Specify the desired IP for the host. 
+
+.. Create the `ClusterDeployment` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f ibi-image-cluster-install.yaml.yaml
+----
+
+. Create the `ClusterDeployment` resource:
+
+.. Create a YAML file that defines the `ClusterDeployment` resource:
++
+.Example `ibi-cluster-deployment.yaml` file
+[source,yaml]
+----
+apiVersion: hive.openshift.io/v1
+kind: ClusterDeployment
+metadata:
+  name: ibi-cluster-deployment <1>
+  namespace: ibi-test
+spec:
+  baseDomain: example.com <2>
+  clusterInstallRef:
+    group: extensions.hive.openshift.io
+    kind: ImageClusterInstall 
+    name: ibi-image-install <3>
+    version: v1alpha1 <4>
+  clusterMetadata:
+    adminKubeconfigSecretRef:
+      name: ibi-test-admin-kubeconfig <5>
+    adminPasswordSecretRef:
+      name: ibi-test-admin-kubeconfig <6>
+    clusterID: da237966-4685-447d-b704-2d4e52a4345c <7>
+    infraID: ibiotest-t4k6t <8>
+  clusterName: ibi-cluster <9>
+  platform:
+    none: {} <10>
+  pullSecretRef:
+    name: pull-secret <11>
+----
+<1> Specify the name for the `ClusterDeployment` resource.
+<2> Specify the base domain that the cluster should belong to.
+<3> Specify the name of the `ImageClusterInstall` in which you defined the container images to use for the image-based installation of the target host.
+<4> What does this do???
```

Comment:
This needs to match the API version of the ImageClusterInstall you're referencing. Technically there can be multiple.

For now we only have the one so you probably don't need a note about it here.

Reply:
Removed note.